### PR TITLE
Problem:(CRO-542) no integration test for hd-wallet

### PR DESCRIPTION
**Pull request by leejw51**
Solution: add hdwallet integration test

### CODE REVIEW

**Comment by leejw51crypto** [comment]
because, hd-wallet is the same with normal wallets except for wallet creation.
this test is enough for CI build.

also other test (mnemonic) is already done in unit test.

File: integration-tests/client-rpc/test/hdwallet-management.test.ts [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+	it("can create wallet with specified name", async () => {
+		const walletName = generateWalletName();
+		const walletRequest = newWalletRequest(walletName, "123456");
+
+		const walletCreateResult = await client.request("wallet_create", [
+			walletRequest
+			, "HD"
+		]);
+		let res= walletCreateResult.split(" ");
+		expect(res.length).to.deep.eq(24);
```
Comment (tomtau) [inline integration-tests/client-rpc/test/hdwallet-management.test.ts]: why not `expect(walletCreateResponse).to.deep.eq(walletName);` ?

Comment (tomtau) [reply integration-tests/client-rpc/test/hdwallet-management.test.ts]: Oh, there are 24 words returned?

Comment (leejw51crypto) [reply integration-tests/client-rpc/test/hdwallet-management.test.ts]: yes

**Comment by leejw51crypto** [comment]
ok, i will add recovery check

**Comment by leejw51** [comment]
because it needs cro-540, i included it.

**Comment by leejw51** [comment]
ok

**Comment by leejw51crypto** [comment]
i'll rebase

**Comment by leejw51crypto** [comment]
let's use viewkey from transfer index=0,
because viewkey is primarily used for hiding utxos

**Pull request merged**